model/map/src/main/java/org/keycloak/models/map/client/MapClientProvider.java

```diff
@@ -152,7 +152,7 @@ public void unregisterNode(String nodeHost) {
         return Stream.concat(tx.createdValuesStream(clientStore.keySet()), updatedAndNotRemovedClientsStream);
     }
 
-//    @Override
+    @Override
```

Comment:
Line 164-167: `public List<ClientModel> getClients(RealmModel realm)` doesn't have to implemented here, right? It's deprecated and used nowhere.

Reply:
removed